.github/actions/action-maven-release/action.yml

```
@@ -0,0 +1,91 @@
+    description: "artifact name from pom"
+    value: ${{ steps.maven-release-step.outputs.MVN_ARTIFACT_ID }}
+  artifact-name:
+    description: "name of the artifact upload"
+    value: ${{steps.artifact-name.outputs.artifact-name}}
+
+runs:
+  using: "composite"
+  steps:
+    # Checkout source code, set up Java, etc. Then...
+    - name: Checkout code
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)